types/index.d.ts

```diff
@@ -0,0 +1 @@
+../src/index.d.ts
```

Comment:
This is a symlink...unfortunately `dtslint` is pretty annoying with it's lack of configurability.

In order to get the tests to run, you need the type definitions and the "tests" to be located in the same directory. Since I didn't want to touch the main `/src` directory, since it's part of the bundling path, I decided to just make a simple symlink here to let the tests run against the main type definitions.

It all works, but I would understand if there was pushback here. Although I do think that git-based symlinks work on every OS nowadays.